test/e2e/create_test.go

```diff
@@ -718,4 +718,21 @@ var _ = Describe("Podman create", func() {
 		setup.WaitWithDefaultTimeout()
 		Expect(setup).Should(Exit(0))
 	})
+
+	It("create container with name subset of existing ID", func() {
```

Comment:
Can you also add a regression test for pods?

Reply:
Done